packages/components/src/components/input/input.stories.ts

```diff
@@ -69,731 +35,219 @@ export const Default = {
+ * Use the `read-only` attribute to render an input as read only.
  */
-
-export const Disabled = {
-  parameters: {
-    controls: {
-      exclude: ['disabled']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
-    return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'disabled', value: true }],
-          args
-        })}
-      </div>
-    `;
-  }
+export const ReadOnly = {
+  render: () =>
+    html`<div class="w-[250px]">
+      <sd-input label="Label" placeholder="Read only example" readonly></sd-input>
+    </div>`
 };
 
 /**
- * Use the readonly attribute to render an input as readonly.  Interaction is enabled, but the input cannot be edited.  Events will be fired.
+ * Use the `help-text` attribute to add a descriptive “help text”. For help texts that contain HTML, use the `help-text` slot instead.
  */
 
-export const Readonly = {
-  parameters: {
-    controls: {
-      exclude: ['readonly']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' }
-  ]),
-  render: (args: any) => {
+export const HelpText = {
+  render: () => {
     return html`
-      <div class="w-[250px]">
-        ${generateTemplate({
-          constants: [{ type: 'attribute', name: 'readonly', value: true }],
-          args
-        })}
+      <div class="flex flex-rows gap-12">
+        <sd-input label="Label" help-text="Help text"></sd-input>
+        <sd-input label="Label">
+          <div slot="help-text" class="text-lg">Help text</div>
+        </sd-input>
       </div>
     `;
   }
 };
 
 /**
- * This shows sd-input in its various sizes.
+ * Add the `clearable` attribute to add a clear button when the input has content.
  */
 
-export const Sizes = {
-  parameters: {
-    controls: {
-      exclude: ['size']
-    }
-  },
-  args: overrideArgs([
-    { type: 'attribute', name: 'value', value: 'value' },
-    { type: 'attribute', name: 'label', value: 'Label' },
-    { type: 'attribute', name: 'help-text', value: 'help-text' },
-    { type: 'attribute', name: 'clearable', value: true },
-    {
-      type: 'slot',
-      name: 'right',
-      value: '<sd-icon slot="right" library="global-resources" name="system/picture"></sd-icon>'
-    }
-  ]),
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
+export const Clearable = {
+  render: () =>
+    html`<div class="w-[250px]">
+      <sd-input label="Clearable" clearable spellcheck></sd-input>
+    </div>`
```

Comment:
Icon is missing if input is empty. So please fill "Input text" inside the input.

![image](https://github.com/user-attachments/assets/c0b6d492-53f1-4e34-bd21-cb81d8339acf)